src/near/index.ts

```diff
@@ -40,16 +42,28 @@ export default class NearWallet implements IWallet {
     this.newAccountAmount = '100000000000000000000000';
   }
 
+  public static getDerivationPath(addressIndex: number, _accountType: string) {
+    return (
+      intToUintByte(5, 8) +
+      intToUintByte(0x80000000 + 44, 8 * 4) +
+      intToUintByte(0x80000000 + 397, 8 * 4) +
+      intToUintByte(0x80000000, 8 * 4) +
+      intToUintByte(0x80000000, 8 * 4) +
+      intToUintByte(0x80000001 + addressIndex, 8 * 4) +
+      intToUintByte(0, 64) // dummy chain id
+    );
+  }
+
   newReceiveAddress(): string {
     return this.address;
   }
 
   public getDerivationPath(sdkVersion: string): string {
     const purposeIndex = '8000002c';
     const coinIndex = this.coin.coinIndex;
+    // TODO: ADD ACCOUNT INDEX HERE
```

Comment:
In near, account index is not incremented. instead level 5 (address index) is incremented.

eg: if first account is `m/44'/397'/0'/0'/1'` then the next is `m/44'/397'/0'/0'/2'`
so accountIndex is always `0x8000000` in near.

```suggestion
```